packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -1343,3 +1352,151 @@ export function listenToEventResponderEventTypes(
 if (enableEventAPI) {
   setListenToResponderEventTypes(listenToEventResponderEventTypes);
 }
+
+const emptyObject = {};
+
+export function createEventTargetHitSlop(
+  left: number | void | null,
+  right: number | void | null,
+  top: number | void | null,
+  bottom: number | void | null,
+  rootContainerInstance: Element | Document,
+  parentNamespace: string,
+): Element {
+  const hitSlopElement = createElement(
+    'div',
+    emptyObject,
+    rootContainerInstance,
+    parentNamespace,
```

Comment:
This won't work if the parentNamespace isn't HTML so you should throw if you don't intend to support other namespaces.